packages/edit-widgets/src/blocks/widget-area/edit/inner-blocks.js

```diff
@@ -40,6 +40,7 @@ export default function WidgetAreaInnerBlocks() {
 
 	return (
 		<div
+			data-widget-area-id={ id }
```

Comment:
Can it go on this div?
https://github.com/WordPress/gutenberg/blob/64f1de6587425808b02a339f82d5fb95193c0011/packages/edit-widgets/src/blocks/widget-area/edit/inner-blocks.js#L50

Otherwise themers will have to do something like:
```css
[data-widget-area-id="my-footer"] div { 
    background-color: cornflowerblue;
}
```

instead of just:
```css
[data-widget-area-id="my-footer"] { 
    background-color: cornflowerblue;
}
```